creativecommons/cccatalog-frontend
# Meta search

## Fixes
Fixes #921 

## Description
Implements legacy meta search for images, audio, and video.

## Preview Url

https://coolsearch.zackkrida.vercel.app

## Testing notes

- In addition to reviewing code, please review the preview url provided with the following steps:
  - Make a search
  - Visit the audio + video tabs and the various external links
  - Scroll to the bottom of the image results and click the "Not finding what you need? Search other sources" link, visit external providers
  - Note: "Use" filters in the sidebar only apply to Europeana, Soundcloud, and Google. Other providers do not offer granular filtering by use case. Please check these filters and make sure they apply as you would expect them to.

## Checklist
 
  index.md`).
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
